src/owr/OptimisticWithdrawalRecipientPectra.sol

```diff
@@ -0,0 +1,451 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import {Clone} from "solady/utils/Clone.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {OwnableRoles} from "solady/auth/OwnableRoles.sol";
+import {SafeTransferLib} from "solady/utils/SafeTransferLib.sol";
+
+/// @title OptimisticWithdrawalRecipientPectra
+/// @author Obol
+/// @notice A maximally-composable contract that distributes payments
+/// based on threshold to it's recipients
+/// @dev Only ETH can be distributed for a given deployment. There is a
+/// recovery method for tokens sent by accident.
+contract OptimisticWithdrawalRecipientPectra is Clone, OwnableRoles {
```

Comment:
```suggestion
contract ControllableWithdrawalRecipient is Clone, OwnableRoles {
```
```suggestion
contract EditableWithdrawalRecipient is Clone, OwnableRoles {
```
```suggestion
contract ControllableOptimisticWithdrawalRecipient is Clone, OwnableRoles {
```

Trying to decide on what name might make the most sense, not convinced people will remember these features by the hardfork they are introduced in. Want to keep the withdrawal recipient suffix. Probably should keep the optimistic piece, to indicate that it optimistically assumes rewards are rewards not worst case slashing. Hoping to incorporate the 'exitable' and 'editable' aspects potentially. 


Have a think and maybe run by me what option we go with before you make the changes.

Reply:
Named it to `OptimisticWithdrawalRecipientV2`